zjit/src/options.rs

```diff
@@ -165,6 +194,19 @@ fn parse_option(options: &mut Options, str_ptr: *const std::os::raw::c_char) ->
 
         ("perf", "") => options.perf = true,
 
+        ("allowed-iseqs", _) if opt_val != "" => options.allowed_iseqs = Some(parse_jit_list(opt_val)),
+        ("save-compiled-iseqs", _) if opt_val != "" => {
```

Comment:
WDYT about calling it `--zjit-log-compiled-iseqs`? This is a compilation log. "save-compiled-iseqs" could sound like a compilation cache, so that'd be confusing when we actually introduce it.

Note that YJIT calls it `--yjit-log`, so it might make sense to call it `--zjit-log` too, which should probably support printing to stderr as well in that case.